ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivCreatePrivacyGroupTest.java

```diff
@@ -60,11 +60,12 @@ public void verifyCreatePrivacyGroup() {
     final String expected = "a wonderful group";
     final PrivacyGroup privacyGroup =
         new PrivacyGroup(expected, PrivacyGroup.Type.PANTHEON, NAME, DESCRIPTION, ADDRESSES);
-    when(enclave.createPrivacyGroup(any(CreatePrivacyGroupRequest.class))).thenReturn(privacyGroup);
+    when(privateTransactionHandler.createPrivacyGroup(ADDRESSES, NAME, DESCRIPTION))
```

Comment:
nit: do you have the energy to change the names of the tests in general? I.e. verifyGroupReportedByPrivacyManagerIsReturnedToCaller?